scripts/setup_integration/setup_integration_env.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -o pipefail
+
+# setup_integration_env.sh
+#
+# SUMMARY
+#
+#  Sets up Vector integration test environments
+
+set -x
+
+# Echo usage if something isn't right.
+usage() {
+    echo "Usage: $0 [-i Name of integration suite ] [-a Action to run {stop|start} ] [-t The container tool to use]" 1>&2; exit 1;
```

Comment:
I would recommend making the non-optional parameters into arguments rather than options, ie:
```suggestion
    echo "Usage: $0 [-t The container tool to use] {stop|start} integration_suite" 1>&2; exit 1;
```

Alternatively, you could build the `stop|start` bit into the filename, by symlinking this script as `start_integration_env.sh` and `stop_integration_env.sh`, and then using that first word of `$0` as the action to take. That's a little sneaky though.